<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="NickNameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;nickname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddFriendButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="FriendListTabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>231</width>
     <height>381</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="HeadPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="IDlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePasswdButton">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change Passwd</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
